### DISCUSSION TITLE
New unit to adjust container height : dvh

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21919] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

If we want the height of the container to be equal to the screen, we use the vh unit. But the problem with this unit is that it ignores the bar address on mobile and makes the page scrollable on mobile.

### Describe the solution you'd like

It would be great if you add the dvh unit. This unit is much more practical than vh. Because it adjusts the height dynamically.

**Comment by 79ho3ein** [comment]
Can use custom unit for container height.

**Reply by FireRedDev** [reply]
doesnt really make sense to offer vh, but not dvh as a normal option, though?
